src/AzureClient/Extensions.cs

```diff
@@ -90,7 +90,6 @@ public static string ToDescription(this AzureClientError azureClientError)
                     {
                         ("JobId", job => job.Id),
                         ("JobStatus", job => job.Status),
-                        ("JobUri", job => job.Uri.ToString()),
```

Comment:
Why is the job's URI no longer included?

Reply:
The `Uri` property of `CloudJob` just throws a `NotImplementedException`: https://github.com/microsoft/qsharp-runtime/blob/4fba91a886ebededb75c17e6d6e05cf08d1e8c97/src/Azure/Azure.Quantum.Client/JobManagement/CloudJob.cs#L63

Reply:
I see, that is one of the tasks that we still have to do in qsharp-runtime. The issue tracking that work is the following: https://github.com/microsoft/qsharp-runtime/issues/236

Should an issue be created in this repo to update this extension once the issue in qsharp-runtime is resolved?

Reply:
Yes, I actually have a task in DevOps to update these extensions for better display. I will add this note to that task and link to the issue you referenced. Thanks!

Reply:
Thanks!